db/schema.rb

```diff
@@ -11,7 +11,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20170110135857) do
+ActiveRecord::Schema.define(version: 20170110185851) do
```

Comment:
I guess that because I have separate some migrations that were originally one file, they were merged with the wrong order because the file named had changed, I have renamed the files so they stay in the correct order